code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -213,3 +213,18 @@
 	key_third_person = "signs"
 	message_param = "signs the number %t."
 	hands_use_check = TRUE
+
+/datum/emote/living/carbon/human/snore
+	key = "snore"
+	key_third_person = "snores"
```

Comment:
Don't define two identical snore emotes, one with a blacklist on living and one for humans only

Just have one snore emote, and implement `get_sound` on it to check 
```dm
if(ishuman(user)) 
    return get_snore_sound()
return null
```